data/transition-sites/esfa_ncs.yml

```diff
@@ -0,0 +1,8 @@
+---
+site: esfa_ncs
+whitehall_slug: education-and-skills-funding-agency
+homepage: http://www.nationalcareers.service.gov.uk
+tna_timestamp: 20201010101010 # Stub timestamp - site not in TNA
+host: nationalcareersservice.direct.gov.uk
+aliases:
```

Comment:
I think they also want `staging.nationalcareersservice.direct.gov.uk` so they can begin to add redirects before transitioning the main domain.

Reply:
I ws going to ask you today about the staging site - does it simply get added as another alias?

Reply:
To be honest I’m not 100% sure. I was trying to find documentation for it but couldn’t find anything specific.

I think it makes sense that we add the alias here because it will then get configured in Fastly and we can configure the DNS for staging only. It seems like it should work in theory.